custom/conf/app.ini.sample

```diff
@@ -210,6 +210,8 @@ PATH = data/gitea.db
 SQLITE_TIMEOUT = 500
 ; For iterate buffer, default is 50
 ITERATE_BUFFER_SIZE = 50
+; Show the database generated SQL
+SHOW_SQL = true
```

Comment:
I think it should better be named `LOG_SQL`

Reply:
done.